tests/brevitas_examples/test_quantize_model.py

```diff
@@ -0,0 +1,305 @@
+        act_bit_width=act_bit_width,
+        bias_bit_width=bias_bit_width,
+        weight_quant_granularity='per_tensor',
+        act_quant_percentile=99.9,
+        act_quant_type='sym',
+        scale_factor_type='float_scale',
+        quant_format='int',
+        layerwise_first_last_bit_width=5,
+    )
+    # Assert it is a GraphModule
+    assert isinstance(quant_model, torch.fx.graph_module.GraphModule)
+
+    # Assert we can feed data of the correct size through the model
+    quant_model(torch.rand(1, 10, IMAGE_DIM, IMAGE_DIM))
+
+    # Get first/last layer for testing its quantization.
+    first_conv_layer = quant_model.get_submodule('0')
+    first_relu_layer = quant_model.get_submodule('1')
+    last_layer = quant_model.get_submodule('6')
+    last_layer_output = quant_model.get_submodule('_6_output_quant')
+
+
+    # Check quantization is toggled as expected
+    assert first_conv_layer.is_weight_quant_enabled
+    assert first_conv_layer.is_bias_quant_enabled
+    assert not first_conv_layer.is_input_quant_enabled # unlike with the layerwise backend, the input quantization is disabled.
+    assert not first_conv_layer.is_output_quant_enabled
+
+    assert not first_relu_layer.is_input_quant_enabled 
+    assert first_relu_layer.is_output_quant_enabled # the output of the "fused" ConvReLU is quantized
+
+    # Assert types are as expected
+    assert isinstance(quant_model.get_submodule('3'), QuantReLU)
+
+    # Assert quantization bit widths are as desired
+    # Biases
+    assert first_conv_layer.bias_quant.tensor_quant.msb_clamp_bit_width_impl.bit_width._buffers['value'].item() == bias_bit_width
+    assert last_layer.bias_quant.tensor_quant.msb_clamp_bit_width_impl.bit_width._buffers['value'].item() == bias_bit_width
+    # Weights
+    assert first_conv_layer.weight_quant.tensor_quant.msb_clamp_bit_width_impl.bit_width._buffers['value'].item() == weight_bit_width
+    assert last_layer.weight_quant.tensor_quant.msb_clamp_bit_width_impl.bit_width._buffers['value'].item() == weight_bit_width
+    # Activations
+    assert first_relu_layer.act_quant.fused_activation_quant_proxy.tensor_quant.msb_clamp_bit_width_impl.bit_width._buffers['value'].item() == act_bit_width
+    assert last_layer_output.act_quant.fused_activation_quant_proxy.tensor_quant.msb_clamp_bit_width_impl.bit_width._buffers['value'].item() == act_bit_width
+
+def test_float_quantization(simple_model):
+    quant_model = quantize_model(
+        model=simple_model,
+        backend='layerwise',
+        weight_bit_width=8,
+        act_bit_width=8,
+        bias_bit_width=32,
+        weight_quant_granularity='per_tensor',
+        act_quant_percentile=99.9,
+        act_quant_type='sym',
+        scale_factor_type='float_scale',
+        quant_format='float',
+    )
+    assert isinstance(quant_model, nn.Sequential)
+    #assert all(isinstance(m, nn.Conv2d) or isinstance(m, nn.Linear) for m in quant_model.children())
+    #TODO
```

Comment:
What is missing here?

Reply:
I'll have to think about it! I left is a reminder I wasn't happy with the test coverage.